package.json

```diff
@@ -20,6 +20,9 @@
     "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
     "create-component:react": "yarn workspace @metamask/design-system-react create-component",
     "create-package": "ts-node scripts/create-package",
+    "generate-icons": "yarn generate-icons:react && yarn generate-icons:react-native",
+    "generate-icons:react": "yarn workspace @metamask/design-system-react generate-icons",
+    "generate-icons:react-native": "yarn workspace @metamask/design-system-react-native generate-icons",
```

Comment:
Added `generate-icons` script to root for quick access. The naming convention follows `create-component:react`. Updated other scripts to `generate-icons` as well